error when running code

index-Beu2SCBU.js:75 Error: TypeError: Cannot read properties of null (reading 'inVideoQuizQuestions')
    at <anonymous>:96:44

<img width="665" height="172" alt="Image" src="https://github.com/user-attachments/assets/c0500318-d388-48e1-b561-ac26fc276bbc" />

when I tried to run the code,

--- a/index.js
+++ b/index.js
@@ -67,24 +67,17 @@ const makeApiRequest = () => {
             },
             query: `
             query GetVideoLesson(
-                $uniqueCode: String!,
-                $subsectionUniqueCode: String!,
-                $moduleUniqueCode: String!,
-                $inVideoQuestionFilter: InVideoQuestionFilter,
+                $uniqueCode: String!, 
+                $subsectionUniqueCode: String!, 
+                $moduleUniqueCode: String!, 
+                $inVideoQuestionFilter: InVideoQuestionFilter, 
                 $inVideoQuestionOrder: [InVideoQuestionOrdering]
-            ) {
+            ) {  
                 videoLesson(filter: {
-                    uniqueCode: $uniqueCode,
-                    subsectionUniqueCode: $subsectionUniqueCode,
+                    uniqueCode: $uniqueCode, 
+                    subsectionUniqueCode: $subsectionUniqueCode, 
                     moduleUniqueCode: $moduleUniqueCode
-                }) {
-                    id
-                    inVideoQuizQuestions(
-                        filter: $inVideoQuestionFilter,
-                        order: $inVideoQuestionOrder
-                    ) {
-                        postQuestionResumeTime
-                    }
+                }) {    id    module {      id      uniqueCode      title      examBoardSubject {        subject: subjectObject {          qualification {            uniqueCode            __typename          }          __typename        }        __typename      }      __typename    }    subsection {      id      name      uniqueCode      __typename    }    wistiaId    uniqueCode    title    length    duration    extraContent    skipRecapStartTime    preVideoQuestions {      __typename      id      quizContent {        ...UnmarkedQuestion        __typename      }    }    inVideoQuizQuestions(      filter: $inVideoQuestionFilter      order: $inVideoQuestionOrder    ) {      __typename      id      triggerTime      postQuestionResumeTime      quizContent {        __typename        ...UnmarkedQuestion      }    }    __typename  }}fragment UnmarkedQuestion on QuizContent {  __typename  id  stem  quizDefinition {    __typename    questions {      ...UnmarkedQuestionPart      __typename    }  }}fragment UnmarkedQuestionPart on QuizQuestion {  __typename  ... on MultipleChoiceQuestion {    question    description    image    topImage    options {      text      image      __typename    }    __typename  }  ... on MultiMultipleChoiceQuestion {    question    description    image    topImage    options {      text      image      __typename    }    __typename  }  ... on TextQuestion {    question    description    image    topImage    beforeText    afterText    __typename  }  ... on NumericalQuestion {    question    description    image    topImage    beforeText    afterText    __typename  }  ... on MathsQuestion {    question    description    image    topImage    __typename  }  ... on MultipleInputQuestion {    questionSegments {      type: __typename      ... on MultipleInputQuestionText {        text        __typename      }      ... on MultipleInputQuestionBlank {        fieldIndex        __typename      }    }    description    image    topImage    __typename  }  ... on ChemistryQuestion {    question    description    image    topImage    __typename  }  ... on DropdownQuestion {    question    description    image    topImage    dropdownOptions    __typename  }  ... on DrawQuestion {    question    description    image    topImage    drawOn    __typename  }  ... on EngageQuestion {    question    description    image    topImage    __typename  
                 }
             }
             `

--- a/index.js
+++ b/index.js
@@ -41,6 +41,9 @@ const makeApiRequest = () => {
     const subsectionUniqueCode = urlParts[5];
     const uniqueCode = urlParts[6];
 
+    const match = document.cookie.match(new RegExp('(^|;\\s*)' + "auth-token" + '=([^;]*)'));
+    const auth = match ? decodeURIComponent(match[2]) : null;
+
     fetch("https://web.uplearn.co.uk/api/", {
         headers: {
             "accept": "*/*",
@@ -51,7 +54,8 @@ const makeApiRequest = () => {
             "sec-ch-ua-platform": "\"Windows\"",
             "sec-fetch-dest": "empty",
             "sec-fetch-mode": "cors",
-            "sec-fetch-site": "same-origin"
+            "sec-fetch-site": "same-origin",
+            "authorization": `Bearer ${auth}`
         },
         credentials: "include",
         body: JSON.stringify({

